### PR TITLE
Done tasks 3.2, 3.3 from batch 03

### PR DESCRIPTION
**Pull request by geka-flash-85**

### CODE REVIEW

File: Yevhenii_Svystunov/03/Array_diff.py [hunk 1]
```diff
@@ -3,13 +3,13 @@ def array_diff(x, y):
     """ Returns difference between two lists.
     
     
-    Parameters:
-        x - is the first list
-        y - is the second list
+        Parameters:
```
Comment (medviediev) [inline Yevhenii_Svystunov/03/Array_diff.py]: wrong params and returns declaration in docstring, rtypes are missing

File: Yevhenii_Svystunov/03/Filter.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            is True, then we write element of source list to this list and so on
+        
+      
+    """  
+
+    new_list = []
+    
+    for i in user_array:    
+        
+        x = user_func(i)
```
Comment (medviediev) [inline Yevhenii_Svystunov/03/Filter.py]: Just if user_func(i) is better

File: Yevhenii_Svystunov/03/Array_diff.py (hunk 1 above)
Comment (geka-flash-85) [reply Yevhenii_Svystunov/03/Array_diff.py]: I think I don't understand how to create docstrings in general so I have the mistakes

File: Yevhenii_Svystunov/03/Filter.py (hunk 2 above)
Comment (geka-flash-85) [reply Yevhenii_Svystunov/03/Filter.py]: Yes, your variant is better

**Pull request merged**